include/nccl_ofi.h

```diff
@@ -439,14 +440,6 @@ struct nccl_net_ofi_plugin {
 	int num_devs;
 };
 
-/**
- * Initialize plugin. This function sets properties of the global plugin variable
- * defined below.
- */
-void nccl_net_ofi_init_plugin(nccl_net_ofi_plugin_t *plugin,
```

Comment:
I think you also need to remove the definition of this function.

Reply:
thanks.